src/internal/FloatingMenu.js

```diff
@@ -149,6 +172,33 @@ class FloatingMenu extends React.Component {
     });
   };
 
+  adjustOffsets(newOffsets, menuDirection) {
+    /**
+     * we have default offsets to center the positions.
+     * right and left positions need further adjustment as follows:
+     * right: top += 3, left += 6
+     * left: top += 3
+     */
+    const defaultOffsets = {
+      left: 5,
+      top: 10,
+    };
+    const adjustedOffsets = Object.assign({}, defaultOffsets);
+    adjustedOffsets.left += newOffsets.left || 0;
+    adjustedOffsets.top += newOffsets.top || 0;
+    switch (menuDirection) {
+      case 'right':
+        adjustedOffsets.left += 8;
+        adjustedOffsets.top += 4;
+        break;
+      case 'left':
+        adjustedOffsets.top += 4;
+        break;
+    }
```

Comment:
I wanted to write this as:
```
switch (menuDirection) {
   case 'right':
     adjustedOffsets.left += 8;
   case 'left':
     adjustedOffsets.top += 4;
 }
```
but your linter wouldn't let me!

Reply:
What was the lint rule violation? @DmitriWolf

Reply:
`error  Expected a 'break' statement before 'case'  no-fallthrough`

Reply:
Definitely will update the lint config to change that, then 😄